dotcom-rendering/src/web/components/Nav/ExpandedMenu/Column.tsx

```diff
@@ -281,6 +295,7 @@ export const Column = ({
 					</li>
 				))}
 			</ul>
+			<div css={[hideStyles(columnInputId), lineStyle]}></div>
```

Comment:
Just to check, `hideStyles()` here is to hide the `lineStyle` applied to this div?